docs/renewables/notebook.rst

```diff
@@ -0,0 +1,15 @@
+Notebooks
++++++++++
+The `RAP notebook`_ fetches the data, imputes missing data and generates a profile.
+
+The `NSRDB SAM notebook`_ generates a solar profile by feeding the NSRDB data to
+SAM while the `NSRDB naive notebook`_ uses the naive method.
```

Comment:
What is 'the naive method'?

Reply:
It is presented under section 6.4.1.2.1:
<img width="827" alt="Screen Shot 2021-07-20 at 10 19 35 PM" src="https://user-images.githubusercontent.com/16549571/126435017-6a3de22a-47a3-4cf0-9f00-6dba5745c16b.png">